setup.py

```diff
@@ -16,7 +16,7 @@
 
 install_requires = [
     'responses',
-    'pytest>=2.5',
+    'pytest>=3.6',
```

Comment:
sentry runs pytest 3.5 right now, our zeus project runs 3.2.x, so requiring a significantly newer version will create headaches for in-house developers

Reply:
(we likely can upgrade those, but that will of course block this PR for us)

Reply:
Please consider either a) handling older versions also (which should be trivial), or b) create a release with an upper version for the pytest dep.

@mixja 
Are you up for adjusting the PR, to still use `get_marker` if `get_closest_marker` is not available?